# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvi/no-catman.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


This catman stuff is stuipid - I don't like it!

 - Clifford

--- ./Makefile~	Mon Oct 26 15:42:21 1998
+++ ./Makefile	Mon Oct 26 15:49:38 1998
@@ -30,7 +30,7 @@
 	install -o root -g bin -m 555 bin/nvi $(BINDIR)/nvi
 	(cd $(BINDIR); ln -sf nvi nex)
 	install -o root -g root -m 444 USD.doc/vi.man/vi.1 $(MANDIR)/$(MANPREFIX)vi.1
-	install -o root -g root -m 444 USD.doc/vi.man/vi.0 $(CATMANDIR)/$(MANPREFIX)vi.1
+#	install -o root -g root -m 444 USD.doc/vi.man/vi.0 $(CATMANDIR)/$(MANPREFIX)vi.1
 
 clean: PORT/linux/Makefile
 	make -C PORT/linux $(MFLAGS) clean
